### ISSUE TITLE
[a11y-defect-2]: Headings are not in sequential order on Review Page

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#71876]
### Point of contact

@jeana-adhoc

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

On the review page, in the `Disability exams detail` box, the heading on the card is an `h5` and in this context should be an `h4`

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/fdf7ea2f-b8c7-411a-ad3e-7e8b40d1c8ae)

### Reproduction steps

1. [Fill out form 20-10206 on staging](https://staging.va.gov/records/request-personal-records-form-20-10206/introduction)
2. Select "Disability examinations (C&P exams)" on the records page
3. On the review page inspect "Disability exam details"
4. See that "exam date" is an `<h5>`

### Proposed solution or next steps

On the review page `exam page` should be initial capped to `Exam page` and it should be an `<h4>`

### References, articles, or WCAG support

1. [WCAG SC 1.3.1_A](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
2. [WCAG SC 2.4.6_AA](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-descriptive.html)
3.
4.
...

### Type of issue

- [X] Axe-core
- [ ] Screenreader | assistive tech & device support
- [X] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [X] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [X] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by tlei123** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27136) opened.

**Comment by tlei123** [comment]
[PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27136) merged &mdash; changes should be in Staging ~4:45 PM ET.

**Comment by tlei123** [comment]
@jeana-adhoc , changes are now [in Staging](https://staging.va.gov/records/request-personal-records-form-20-10206), ready for validation.

**Comment by jeana-adhoc** [comment]
Confirmed fixed in staging!

<img width="650" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/44112a1c-045f-4762-a8d4-582958b30383">
